Pre_processing.py

```
@@ -15,24 +15,18 @@
-        #Give the dataframe some column headers
-        column_names = ["target", "id", "date", "flag", "user", "text"]
-        df_out = df.toDF(*column_names)
+    def load_data(self,data_location):
+        df_out = spark.table(data_location)
         return df_out
 
-    def filter_df(self,df):
+    def filter_data(self,df):
         #Filter out the columns we need
-        df_out = df.select("target","text")
+        df_out = df[["sentiment","text"]].orderBy(rand()).limit(10000)
```

Ruff

Undefined name `rand`

(F821)